tools/topology/topology2/include/pipelines/cavs/volume-capture.conf

```diff
@@ -0,0 +1,69 @@
+#
+# CAVS volume capture pipeline
+#
+# A simple volume pipeline. All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.volume-capture.N.attribute_name".
+#
+# Usage: volume-capture pipeline object can be instantiated as:
+#
+# Object.Pipeline.volume-capture."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+Class.Pipeline."volume-capture" {
```

Comment:
@RanderWang this name will clash with the volume-cpature we use in SOF no? Can we use a different name

Reply:
yes. actually there is no module called volume in ipc4 and it is gain module. It is named to volume to let us easy to get it.  Maybe gain is more suitable for ipc4